lib/modules/userTagger.js

```diff
@@ -506,14 +507,11 @@ modules['userTagger'] = {
 					taggedUsers.reverse();
 				}
 				break;
+			case 'username':
+				modules['userTagger'].sortUsername(taggedUsers, this.descending);
```

Comment:
may as well  simply `case 'username': // falls through` and delete the rest of this case.